docs/docs/download.md

```diff
@@ -32,3 +32,6 @@ Run the command below.
 npm install -g mbt
 ```
 
+From MBT v1.2.25, build-parameters ignore attribute will be upgraded to support [Full Glob Pattern](https://en.wikipedia.org/wiki/Glob_(programming)) which need to install micromatch-wrapper.
```

Comment:
From MBT v1.2.25, the 'build-parameters ignore' attribute will be upgraded to support [Glob patterns](https://en.wikipedia.org/wiki/Glob_(programming)) which need to install 'micromatch-wrapper'.
(I don't understand what is meant at the end of the sentence- which need to install. Maybe "You must install the 'micromatch-wrapper' to use the Glob patterns.

Reply:
yes,

still use “full glob pattern”